rpcs3/Emu/RSX/GL/GLHelpers.h

```diff
@@ -2692,8 +2721,11 @@ namespace gl
 
 			void remove()
 			{
-				glDeleteProgram(m_id);
-				m_id = 0;
+				if (m_id != GL_NONE)
```

Comment:
I am unsure if uniforms.clear() should be included inside this if check, from my understanding it should be outside

Reply:
It's a generic map, no need to clear inside if check.